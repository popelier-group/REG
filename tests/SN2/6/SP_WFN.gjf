# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.46277000   -0.00007500    0.00022700
 H                 -0.75972300   -0.88133200   -0.54865500
 H                 -0.75980200    0.91590100   -0.48847000
 H                 -0.75957900   -0.03483100    1.03790400
 Br                 1.58767800    0.00000000   -0.00000200
 Cl                -3.35861000    0.00003200   -0.00009600

frame.wfn
