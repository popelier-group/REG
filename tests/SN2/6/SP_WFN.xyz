
6

C       -0.350677    0.008296    0.053597
H       -0.523898   -0.536738    0.969565
H       -0.793614   -0.470334   -0.807092
H       -0.614879    1.052333    0.132721
Br        1.675756   -0.038603   -0.255768
Cl       -3.212648    0.073886    0.490301
