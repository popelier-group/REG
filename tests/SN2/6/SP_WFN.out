 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.6/Gau-15696.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.6/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     15697.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.46277  -0.00008   0.00023 
 H                    -0.75972  -0.88133  -0.54866 
 H                    -0.7598    0.9159   -0.48847 
 H                    -0.75958  -0.03483   1.0379 
 Br                    1.58768   0.       0. 
 Cl                   -3.35861   0.00003  -0.0001 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.462770   -0.000075    0.000227
      2          1           0       -0.759723   -0.881332   -0.548655
      3          1           0       -0.759802    0.915901   -0.488470
      4          1           0       -0.759579   -0.034831    1.037904
      5         35           0        1.587678    0.000000   -0.000002
      6         17           0       -3.358610    0.000032   -0.000096
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.079846   0.000000
     3  H    1.079845   1.798240   0.000000
     4  H    1.079851   1.798258   1.798252   0.000000
     5  Br   2.050448   2.566721   2.566737   2.566725   0.000000
     6  Cl   2.895840   2.798559   2.798415   2.798861   4.946288
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.350677    0.008296    0.053597
      2          1           0       -0.523898   -0.536738    0.969565
      3          1           0       -0.793614   -0.470334   -0.807092
      4          1           0       -0.614879    1.052333    0.132721
      5         35           0        1.675756   -0.038603   -0.255768
      6         17           0       -3.212648    0.073886    0.490301
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         155.0713933           0.8432354           0.8432353
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       177.4863360494 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  9.01D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39403246     A.U. after   12 cycles
            NFock= 12  Conv=0.14D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.68231-101.26022 -62.33535 -56.15513 -56.15205
 Alpha  occ. eigenvalues --  -56.15205 -10.07735  -9.17347  -8.54470  -6.93218
 Alpha  occ. eigenvalues --   -6.93151  -6.93151  -6.37426  -6.36407  -6.36407
 Alpha  occ. eigenvalues --   -2.48545  -2.48257  -2.48257  -2.47449  -2.47449
 Alpha  occ. eigenvalues --   -0.65463  -0.53841  -0.51055  -0.29828  -0.29827
 Alpha  occ. eigenvalues --   -0.23584  -0.15191  -0.15191  -0.06089  -0.06013
 Alpha  occ. eigenvalues --   -0.06013
 Alpha virt. eigenvalues --    0.10070   0.11119   0.14224   0.14224   0.16283
 Alpha virt. eigenvalues --    0.17485   0.18119   0.18120   0.21451   0.22026
 Alpha virt. eigenvalues --    0.22027   0.24318   0.24319   0.24736   0.31256
 Alpha virt. eigenvalues --    0.31257   0.31761   0.34395   0.40655   0.50437
 Alpha virt. eigenvalues --    0.52619   0.52619   0.54313   0.54313   0.67179
 Alpha virt. eigenvalues --    0.67590   0.67591   0.68964   0.75085   0.75085
 Alpha virt. eigenvalues --    0.76032   0.77711   0.77711   0.83041   0.83041
 Alpha virt. eigenvalues --    0.84563   0.95426   1.08910   1.14541   1.14541
 Alpha virt. eigenvalues --    1.15062   1.15063   1.16266   1.23507   1.23508
 Alpha virt. eigenvalues --    1.27376   1.56619   1.67592   1.67593   1.81140
 Alpha virt. eigenvalues --    1.85449   1.85449   2.07849   2.12919   2.12920
 Alpha virt. eigenvalues --    2.29523   2.30839   2.30839   2.52141   2.52143
 Alpha virt. eigenvalues --    2.57082   2.60975   2.64667   2.64668   2.69269
 Alpha virt. eigenvalues --    2.69269   3.12656   3.16592   3.86032   3.86032
 Alpha virt. eigenvalues --    4.28390   4.28390   4.37329   4.37329   4.65891
 Alpha virt. eigenvalues --    6.74033   7.69781   7.69781   7.80174   9.88201
 Alpha virt. eigenvalues --   24.04689  26.08198  26.08200  26.22490  48.03400
 Alpha virt. eigenvalues --  215.84992 289.89793 289.89793 290.018701020.81236
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.071118   0.413155   0.413166   0.413129   0.091726  -0.063921
     2  H    0.413155   0.530209  -0.035520  -0.035510  -0.056635   0.005135
     3  H    0.413166  -0.035520   0.530188  -0.035514  -0.056631   0.005135
     4  H    0.413129  -0.035510  -0.035514   0.530252  -0.056637   0.005138
     5  Br   0.091726  -0.056635  -0.056631  -0.056637  35.394414   0.008353
     6  Cl  -0.063921   0.005135   0.005135   0.005138   0.008353  17.914680
 Mulliken charges:
               1
     1  C   -0.338373
     2  H    0.179165
     3  H    0.179176
     4  H    0.179142
     5  Br  -0.324590
     6  Cl  -0.874520
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.199110
     5  Br  -0.324590
     6  Cl  -0.874520
 Electronic spatial extent (au):  <R**2>=           1155.7909
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             10.0822    Y=             -0.2315    Z=             -1.5386  Tot=             10.2015
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -94.3596   YY=            -43.8060   ZZ=            -44.9573
   XY=              1.1633   XZ=              7.7194   YZ=             -0.1775
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -33.3186   YY=             17.2350   ZZ=             16.0837
   XY=              1.1633   XZ=              7.7194   YZ=             -0.1775
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            214.3484  YYY=             -0.6530  ZZZ=            -10.7671  XYY=             22.2944
  XXY=             -3.9301  XXZ=            -25.9514  XZZ=             25.5912  YZZ=             -1.4507
  YYZ=             -3.3035  XYZ=              0.3897
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1916.8403 YYYY=            -70.7144 ZZZZ=           -102.9489 XXXY=             26.3941
 XXXZ=            174.8945 YYYX=             15.6049 YYYZ=             -2.3773 ZZZX=            108.6033
 ZZZY=             -2.7372 XXYY=           -257.3931 XXZZ=           -272.7582 YYZZ=            -28.9192
 XXYZ=             -2.2277 YYXZ=             35.6386 ZZXY=              6.1097
 N-N= 1.774863360494D+02 E-N=-7.706478223360D+03  KE= 3.070014440939D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE529\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.46277,-0.000075,0.000227\H,0,-0.759723,-0.881332,-0.548655\H,0,
 -0.759802,0.915901,-0.48847\H,0,-0.759579,-0.034831,1.037904\Br,0,1.58
 7678,0.,-0.000002\Cl,0,-3.35861,0.000032,-0.000096\\Version=ES64L-G16R
 evC.01\State=1-A\HF=-3074.3940325\RMSD=1.387e-09\Dipole=4.013582,-0.00
 00862,0.0002624\Quadrupole=-25.667374,12.8336928,12.8336812,0.0003893,
 -0.0012243,-0.0000001\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 IF MATHEMATICALLY YOU END UP WITH THE INCORRECT ANSWER,
 TRY MULTIPLYING BY THE PAGE NUMBER.
 Job cpu time:       0 days  0 hours  0 minutes 26.6 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 13.6 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:05 2020.
